buildstockbatch/aws/aws.py

```diff
@@ -1275,36 +1271,7 @@ def run_job(cls, job_id, bucket, prefix, job_name, region):
         weather_dir = sim_dir / "weather"
         os.makedirs(weather_dir, exist_ok=True)
 
-        # Make a lookup of which parameter points to the weather file from options_lookup.tsv
-        with open(sim_dir / "lib" / "resources" / "options_lookup.tsv", "r", encoding="utf-8") as f:
-            tsv_reader = csv.reader(f, delimiter="\t")
-            next(tsv_reader)  # skip headers
-            param_name = None
-            epws_by_option = {}
-            for row in tsv_reader:
-                row_has_epw = [x.endswith(".epw") for x in row[2:]]
-                if sum(row_has_epw):
-                    if row[0] != param_name and param_name is not None:
-                        raise RuntimeError(
-                            f"The epw files are specified in options_lookup.tsv under more than one parameter type: {param_name}, {row[0]}"
-                        )  # noqa: E501
-                    epw_filename = row[row_has_epw.index(True) + 2].split("=")[1].split("/")[-1]
-                    param_name = row[0]
-                    option_name = row[1]
-                    epws_by_option[option_name] = epw_filename
-
-        # Look through the buildstock.csv to find the appropriate location and epw
-        epws_to_download = set()
-        building_ids = [x[0] for x in jobs_d["batch"]]
-        with open(
-            sim_dir / "lib" / "housing_characteristics" / "buildstock.csv",
-            "r",
-            encoding="utf-8",
-        ) as f:
-            csv_reader = csv.DictReader(f)
-            for row in csv_reader:
-                if int(row["Building"]) in building_ids:
-                    epws_to_download.add(epws_by_option[row[param_name]])
+        epws_to_download = docker_base.determine_weather_files_needed_for_job(sim_dir, jobs_d)
```

Comment:
The logic here is largely unchanged, just moved into docker_base.py.

The one change is that I added the `.ddy`, `.stat`, and `empty.*` files that ComStock seems to want. (Based on [here](https://github.com/NREL/ComStock/blob/61f4bcb4d6355d0819094c6cd4da0180bfdd6d13/resources/measures/ChangeBuildingLocation/measure.rb#L256), [here](https://github.com/NREL/ComStock/blob/61f4bcb4d6355d0819094c6cd4da0180bfdd6d13/resources/measures/ChangeBuildingLocation/measure.rb#L226), and [here](https://github.com/NREL/ComStock/blob/61f4bcb4d6355d0819094c6cd4da0180bfdd6d13/comstock_hpc_training.md?plain=1#L54).)